client/src/components/QandA/QuestionList.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import QuestionListEntry from './QuestionListEntry.jsx';
+import AddQuestion from './AddQuestion.jsx';
+
+const QuestionList = (props) => {
+  return (
+    <div className="container">
+      <p className="text-start fs-5 fw-light">
+      Questions and Answers
+      </p>
+      <div className="input-group mb-3">
+        <input type="text" className="form-control" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..." aria-label="Recipient's username" aria-describedby="button-addon2"></input>
+        <button className="btn btn-outline-secondary" type="button" id="button-addon2">Filter Questions</button>
+        {/* the above will be replaced by a magnifiying glass icon */}
```

Comment:
I think  you might want to consider making this component stateful in order to store the input on the searchbar. When the input is updated, you could make a call to a function passed down from parent component that could change the .filter

Reply:
That's what I was considering doing next! If I can ever get around to finishing my 'load more questions' functionality.